### PR TITLE
Added a page listing dependency versions for each Rancher Desktop release

### PR DESCRIPTION
**Pull request by gunamata**
Signed-off-by: Gunasekhar Matamalam <gunasekhar.matamalam@suse.com>

### CODE REVIEW

File: docs/references/dependency-versions.mdx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: Dependency Versions
+---
+
+import DependencyVersions from '@site/src/components/DependencyVersions';
+
+Rancher Desktop uses several utilities/subsystems as dependencies under the hood for various purposes, for example, **docker CLI** to interact with **dockerd**, **helm** to manage charts, **trivy** for container image scanning, etc. This page provides information about the versions of the dependencies that go into a specific Rancher Desktop release version.
```
Comment (btat) [inline docs/references/dependency-versions.mdx:7]:
```suggestion
Rancher Desktop uses several utilities/subsystems as dependencies under the hood for various purposes. For example, **docker CLI** to interact with **dockerd**, **helm** to manage charts, **trivy** for container image scanning, etc. This page provides information about the versions of the dependencies that go into a specific Rancher Desktop release version.
```

**Review by btat — CHANGES_REQUESTED** [review]

File: docs/references/dependency-versions.mdx (hunk 1 above)
Comment (gunamata) [reply docs/references/dependency-versions.mdx]: Updated the file per your suggestion @btat

**Comment by gunamata** [comment]
@jandubois , I have changed the approach based on your feedback. We just show the dependencies for few utilities that users care about, and this approach involves minor manual updates to docs as part of the release process. I will capture the steps in the releases process doc on Confluence.

**Review by jandubois — APPROVED** [review]
Thanks, LGTM

**Pull request merged**